app/src/main/java/org/sopt/and/MainActivity.kt

```diff
@@ -19,29 +20,11 @@ class MainActivity : ComponentActivity() {
         enableEdgeToEdge()
         setContent {
             ANDANDROIDTheme {
-                Scaffold(modifier = Modifier.fillMaxSize()) { innerPadding ->
-                    Greeting(
-                        name = "Android",
-                        modifier = Modifier.padding(innerPadding)
-                    )
+                val intent = Intent(this, SignUpActivity::class.java).apply {
+                    flags = Intent.FLAG_ACTIVITY_CLEAR_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
                 }
+                startActivity(intent)
```

Comment:
플래그 사용 너무 좋아요 ㅎㅎㅎ 플래그로 스택 관리 잘하면 진짜 유용하답니당
또한 저는 intent 변수를 따로 생성하지 않는 편입니다! 뷰 이동이 많은 인텐트의 경우, 지역 변수이지만 intent라는 변수가 너무 많아지면서 혼동의 여지를 주는 것 같아요.
```suggestion
                startActivity(
                        Intent(this, SignUpActivity::class.java).apply {
                            flags = Intent.FLAG_ACTIVITY_CLEAR_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
                    }
                )
```